etna/models/sarimax.py

```diff
@@ -212,6 +214,126 @@ def get_model(self) -> SARIMAXResultsWrapper:
         """
         return self._fit_results
 
+    @staticmethod
+    def _prepare_components_df(components: np.ndarray, model: SARIMAX) -> pd.DataFrame:
```

Comment:
Is `SARIMAX` a correct typing as we also expect to use it for `AutoSARIMAX`?

Reply:
Yes,  `pmdarima.auto_arima` fits SARIMAX model